app/pages/admin/project-status.jsx

```diff
@@ -73,13 +82,43 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  handleDialogCancel() {
+    this.setState({ dialogIsOpen: false });
+  }
+
+  handleDialogSuccess(workflow) {
+    const promises = [
+      workflow.update({ active: false }).save(),
+      this.state.project.update({ 'configuration.default_workflow': undefined }).save()
+    ];
+
+    Promise.all(promises)
+      .then(() => {
+        this.setState({ dialogIsOpen: false });
+        this.getProjectAndWorkflows();
+      })
+      .catch(error => {
+        this.setState({
+          dialogIsOpen: false,
+          error
+        });
+      });
+  }
+
   handleToggle(event, workflow) {
     this.setState({ error: null });
-    const checked = event.target.checked;
+    const isChecked = event.target.checked;
+    const defaultWorkflowId = this.state.project.configuration.default_workflow;
 
-    return workflow.update({ 'active': checked }).save()
-      .then(() => this.getWorkflows())
-      .catch(error => this.setState({ error }))
+    if (defaultWorkflowId === workflow.id && workflow.active) {
+      this.setState({ dialogIsOpen: true });
+    }
+
+    if (defaultWorkflowId !== workflow.id) {
```

Comment:
I went back and looked at the lab code and if a project isn't live, then the default workflow button can still be enabled for inactive workflows. I think this should change, but to make sure we still allow admins to toggle the checkbox as needed, this conditional needs modification:
`((defaultWorkflowId !== workflow.id) || (defaultWorkflowId === workflow.id && !workflow.active))`